docs/api/v2/node.md

```diff
@@ -8,20 +8,54 @@ TBD.
 
 *NOTE: Some of these are currently at the protocol layer rather than than the Node itself.*
 
-```
+### WakuSub API
+
+```Nim
 method publish*(w: WakuSub, topic: string, data: seq[byte]) {.async.}
 method subscribe*(w: WakuSub, topic: string, handler: TopicHandler) {.async.}
+method unsubscribe*(w: WakuSub, topics: seq[TopicPair]) {.async.}
```

Comment:
I just took this from libp2p, its the `topic` + `TopicHandler` I believe. Just added it so it would be possible to unsubscribe.
Imo, this API should probably be internal only.